.templates/go/default.mk

```diff
@@ -30,23 +42,43 @@ endif
 .deps:
 	touch $@
 
-dist: $(EXE)
+dist: $(EXE) 
 ifndef NO_UPX_COMPRESSION
 	make .dist-compressed
 endif
 	touch $@
 
-$(EXE): .deps $(GO_SOURCE_FILES)
-	mkdir -p dist
+# Define build $(EXE) target differentially depending on NO_CROSS_COMPILE flag
 ifndef NO_CROSS_COMPILE
-	# Cross-compile executable for many platforms
-	go get github.com/aslakhellesoy/gox
-	gox -buildmode=exe -ldflags $(GOX_LDFLAGS) -output "dist/$(EXE_BASE_NAME)-{{.OS}}-{{.Arch}}" -rebuild ./cmd
+$(EXE): $(PLATFORMS)
+	# Rename windows compilations to include .exe extension
+	rm -f dist/*windows*.exe
+	for f in dist/*windows*; do mv -f $${f} $${f}.exe; done
 else
+$(EXE): .deps $(GO_SOURCE_FILES)
 	# Compile executable for the local platform only
 	go build -ldflags $(GOX_LDFLAGS) -o $@ ./cmd
 endif
 
+$(PLATFORMS): .deps $(GO_SOURCE_FILES) supported-go-version
+	mkdir -p dist
+	# Cross-compile executable for many platforms
+	echo "Building $(X-OS)-$(X-ARCH)"
+	GOOS=$(X-OS) GOARCH=$(X-ARCH) go build -buildmode=exe -ldflags $(GOX_LDFLAGS) -o "dist/$(EXE_BASE_NAME)-$(X-OS)-$(X-ARCH)" -a ./cmd
```

Comment:
Let's rename `GOX_LDFLAGS` to `LDFLAGS` now that we're no longer using gox.

Reply:
Adjusted terminology to adjust LDFLAG name convention